installer/project/components/params.xml

```diff
@@ -602,7 +602,7 @@ The installer will generate a script that can be run on the database server to c
           <description>Database Name:</description>
           <explanation></explanation>
           <value></value>
-          <default>odk_prod</default>
```

Comment:
You are missing another `odk_prod` in installer/project/components/configure_war.xml. 

I catch these by always using [ripgrep](https://github.com/BurntSushi/ripgrep) when I'm making these sorts of changes.

Reply:
I think you were looking to a different commit... That line is already fixed